docs/README.md

```diff
@@ -4,18 +4,18 @@
 
 _Modern Firefox Telemetry for mobile platforms_
 
-`Glean` is a modern approach for a mobile Telemetry library.
+The `Glean SDK` is a modern approach for a mobile Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
```

Comment:
[This is the very first people land on](https://mozilla.github.io/glean/), both from the A-C repository as well from the material for our demo and talk next week. Maybe we should put the "Glean overview" document first?

Reply:
Also [bug 1559133](https://bugzilla.mozilla.org/show_bug.cgi?id=1559133) about making this more clear.